lib/crawl.js

```diff
@@ -138,6 +150,9 @@ function crawl(url, flags) {
             });
             urls = limitSimilar(urls, flags.limitSimilar);
         }
+
+        urls = processOverrides(urls, overriddenScenarios);
```

Comment:
This should always be processed? Or should this be processed if `flag.template` exists?

Reply:
Should always. They might have overriden `backstop.template.json`.